Mathlib/Tactic/Linter/MinImports.lean

```diff
@@ -72,26 +72,21 @@ def minImportsLinter : Linter where run := withSetOptionIn fun stx => do
       let currentlyUnneededImports := explicitImportsInFile.diff importsSoFar
       -- we read the current file, to do a custom parsing of the imports:
       -- this is a hack to obtain some `Syntax` information for the `import X` commands
-      let fil ← IO.FS.readFile (← getFileName)
+      let filePath ← getFileName
+      let fil ← IO.FS.readFile filePath
```

Comment:
I renamed `filePath` to `fname` as this is the name of the variable that `IO.FS.readFile` expects.

I used `contents` instead of `fil`, since that seems the most common name used for the output of `readFile` command.  Other options were `text`, `content`, `input`.